mailviews/previews.py

```diff
@@ -13,8 +13,19 @@
 from django.core.urlresolvers import reverse
 from django.http import Http404
 from django.shortcuts import render
-from django.utils.datastructures import SortedDict
-from django.utils.importlib import import_module
+
+try:
+    # Django <= 1.8
+    from django.utils.datastructures import SortedDict as OrganizedDict
+except ImportError:
+    from collections import OrderedDict as OrganizedDict
```

Comment:
No, i don't have any.
Line 21 only fails if Python version is less than 2.7
Now i change it.